boards/arm/mimxrt1062_fmurt6/doc/index.rst

```diff
@@ -0,0 +1,402 @@
++-----------+------------+-------------------------------------+
+| GPT       | on-chip    | gpt                                 |
++-----------+------------+-------------------------------------+
+| TRNG      | on-chip    | entropy                             |
++-----------+------------+-------------------------------------+
+| FLEXSPI   | on-chip    | flash programming                   |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+``boards/arm/mimxrt1062_fmurt6/mimxrt1062_fmurt6_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and I/Os
+====================
+
+The MIMXRT1062 SoC has five pairs of pinmux/gpio controllers.
+
++---------------+-----------------+---------------------------+
+| Name          | Function        | Usage                     |
++===============+=================+===========================+
+| GPIO_AD_B0_00 | LPSPI1_SCK      | GPIO                      |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_01 | LPSPI1_SDO      | SPI                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_02 | FLEXCAN         | CAN                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_03 | LPSPI3_PCS0     | SPI                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_05 | GPIO            | SD Card                   |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_09 | GPIO/ENET_RST   | PWM                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_10 | GPIO/ENET_INT   | GPIO/Ethernet             |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_11 | GPIO            | Touch Interrupt           |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_12 | LPUART1_TX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_13 | LPUART1_RX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B1_00 | LPI2C1_SCL      | I2C                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B1_01 | LPI2C1_SDA      | I2C                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B1_06 | LPUART3_TX      | UART BT HCI               |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B1_07 | LPUART3_RX      | UART BT HCI               |
++---------------+-----------------+---------------------------+
+| WAKEUP        | GPIO            | SW0                       |
++---------------+-----------------+---------------------------+
+| GPIO_B0_00    | LCD_CLK         | LCD Display               |
```

Comment:
Are all these pins accurate? I'm asking because it does not seem like this board has display support, and this pinmux description should describe the default routing of pins on the board

Reply:
Thanks for pointing out.. I am making precise changes now

Reply:
Made the changes as suggested

Reply:
Changes have been done